modules/perception/obstacle/onboard/camera_process_subnode.cc

```diff
@@ -91,37 +91,35 @@ bool CameraProcessSubnode::InitModules() {
 }
 
 void CameraProcessSubnode::ImgCallback(const sensor_msgs::Image &message) {
-  AdapterManager::Observe();
-  sensor_msgs::Image msg = AdapterManager::GetImageFront()->GetLatestObserved();
-
-  double timestamp = msg.header.stamp.toSec();
-
+  double timestamp = message.header.stamp.toSec();
+  ADEBUG << "CameraProcessSubnode ImgCallback: "
+         << " frame: " << ++seq_num_ << " timestamp: ";
+  ADEBUG << std::fixed << std::setprecision(64) << timestamp;
 
   if (FLAGS_skip_camera_frame) {
-      if (timestamp_ns_ > 0.0) {
-        double curr_timestamp = timestamp * 1e9;
-
-        if ((curr_timestamp - timestamp_ns_) < (1e9 / FLAGS_camera_hz)) {
-            AINFO << "CameraProcessSubnode Skip frame";
-            return;
-        }
-        timestamp_ns_ = curr_timestamp;
-      } else {
-        timestamp_ns_ = timestamp * 1e9;
+    if (timestamp_ns_ > 0.0) {
+      double curr_timestamp = timestamp * 1e9;
+
+      if ((curr_timestamp - timestamp_ns_) < (1e9 / FLAGS_camera_hz)) {
+        return;
```

Comment:
I think a ADEBUG here for skipping frame is good, in case later we need it

Reply:
Thanks. Fixed.